### ISSUE TITLE
Modelling collapse with compas_fea?

### ISSUE THREAD

**Issue by Voulp** [compas-dev/compas_fea#113]
Hi,
I have been browsing the tutorial and the examples of compas_fea for Abaqus (https://compas.dev/compas_fea/tutorial.html) looking for options on
- An explicit dynamic step
- General contact options
- Detailed plasticity and failure behaviour of connector elements
However I don't see any of the above (although Andrew has some topics from 2017 which are empty (?).
In the Analysis part of the compas_fea tutorial there is sentence possibly implying that there are many options provided within compas_fea for writing a complex .inp file for Abaqus ("Because the official reference documentation for Abaqus .inp files is vast, it will not be described here what every line of the .inp file means").

So basically can I use compas_fea to use any features of Abaqus, or is it limited to a library within compas_fea? If limited, where can I find this library? The tutorial is a bit vague.

If there is a way to analyse very complex Abaqus models via compas_fea that would be amazing for collapse research.
If not, then is there a way to only import geometry from Rhino to Abaqus and then continue the work inside Abaqus?

Thank you in advance,

Konstantinos

**Comment by brgcode** [comment]
hi konstantinos,

this is currently not possible with FEA(1) but very likely be available in FEA2
